castor.php

```diff
@@ -3,9 +3,10 @@
 use Castor\Attribute\AsTask;
 
 use function Castor\import;
+use function Castor\mount;
 
 import(__DIR__ . '/examples');
-import(__DIR__ . '/tools/phar/castor.php');
+mount(__DIR__ . '/tools/phar', 'castor:phar');
```

Comment:
Why not 

```
import(__DIR__ . '/tools/phar', namespace: 'castor:phar');
```

instead ?

Reply:
I don't think it's a bad idea to separate the two ways of doing things.

`import()` will allow us to add functionalities, and use them with our context.

`mount()` allows you to work in a directory other than root (independently of the main context).

If I've understood correctly the PR ^^

Reply:
@joelwurtz because it's not the same semantics at all.
The mount change the working directory on mount another castor appli. Not juste a file